docs/source/installing_dallinger_for_users.rst

```diff
@@ -91,3 +91,31 @@ you can move on.
 
 Next, you'll need :doc:`access keys for AWS, Heroku,
 etc. <aws_etc_keys>`.
+
+
+Install Heroku
+^^^^^^^^^^^^^^
+
+To run experiments locally or on the internet you need Heroku installed.
+You need at least version 3.28.0 to use Dallinger. You only need
+a Heroku account to launch experiments on the internet, you do not need it
+for local debugging.
+
+You can check your currently installed version by running:
+
+::
+
+    heroku --version
+
+Heroku is available for download from 
+`heroku.com <https://devcenter.heroku.com/articles/heroku-cli>`__.
+
+Install redis
+^^^^^^^^^^^^^
+
+Accurate debugging of experiments also requires you to have redis
```

Comment:
The problem here is that on almost every system the dallinger user won't have permission to start redis or postures, and there would be a lot of operating system specific code. Even on common operating systems there can be two or three ways or starting the service depending in how it was installed.